css/style-variables.css

```diff
@@ -0,0 +1,40 @@
+/* Color Variables */
+:root {
+    /* colors */
+    --white: #ffffff;
+    --black: #000000;
+    --iris: #4D5AE5;
+    --ocean: #404BBF;
+    --navy-blue: #2E2F42;
+    --green: #31D0AA;
+    --slate: #434455;
+    --light-slate: #8E8F99;
+    --cornflower: #E7E9FC;
+    --cloud: #F4F4FD;
+    --navy-blue-modal: #2e2f4266;
+    --grey: #2e2f42b3;
+    --dairy: #FCFCFC;
+
```

Comment:
It is a good practice to apply alphabet sorting in such cases

Reply:
Thank you for the comment. I will definitely take this into account when working on variables file.